src/main/resources/application.yaml

```diff
@@ -148,6 +148,9 @@ security:
       - "uploadHomeOfficeAppealResponse"
       - "uploadAdditionalEvidenceHomeOffice"
       - "uploadAddendumEvidenceHomeOffice"
+    caseworker-ia-iacjudge:
+      - "endAppeal"
+      - "changeDirectionDueDate"
```

Comment:
Could you add a functional tests for changeDirectionDueDate by judge?

Reply:
Added